docs/spec.md

```diff
@@ -468,9 +468,9 @@ tensor. Depending on the element type, does the following:
 
 #### Inputs
 
-| Name      | Type                                                      | Constraints |
-|-----------|-----------------------------------------------------------|-------------|
-| `operand` | tensor of signed integer, floating-point, or complex type | (C1), (C2)  |
+| Label | Name      | Type                                                      | Constraints |
+|---|-----------|-----------------------------------------------------------|-------------|
+ | (I1) | `operand` | tensor of signed integer, floating-point, or complex type | (C1), (C2)  |
```

Comment:
The tables look a bit wonky now: 1) the second row should be extended from 3 hyphens to 6 hyphens in the first column, 2) subsequent rows shouldn't start with a whitespace.

Reply:
Ah, my bad. Fixed.